submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/model/DoiRequest.kt

```diff
@@ -0,0 +1,92 @@
+package ac.uk.ebi.biostd.submission.model
+
+import ebi.ac.uk.util.collections.ifNotEmpty
+import org.redundent.kotlin.xml.PrintOptions
+import org.redundent.kotlin.xml.xml
+import java.time.Instant
+
+data class Contributor(
+    val name: String,
+    val surname: String,
+    val affiliation: String,
+    val orcid: String?
+)
+
+class DoiRequest(
+    private val accNo: String,
+    private val title: String,
+    private val instanceUrl: String,
+    private val contributors: List<Contributor>,
+) {
+    val doi: String
+        get() = "$BS_DOI_ID/$accNo"
+
+    fun asXmlRequest(): String {
+        val timestamp = Instant.now().epochSecond.toString()
```

Comment:
I would move this as a class parameter, fundamentally because I think this class purpose serialize into xml what class called provide, not to calcualte timestamp or other values.